scripts/globals/mobskills/granite_skin.lua

```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- Granite Skin
+-- Description: Physical Shield
+-- TODO: Verify power, messaging
+-- https://www.bg-wiki.com/ffxi/Locus_Wivre
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    xi.mobskills.mobBuffMove(mob, xi.effect.PHYSICAL_SHIELD, 3, 0, 30)
```

Comment:
Granite Skin is a buff that makes targets in front of it deal 0 physical damage. This will need a new effect and some kind of check to zero it properly. I think this is beyond the scope of this PR but it does need to be noted.